tensorflow/contrib/tensorrt/python/trt_convert.py

```diff
@@ -168,6 +176,11 @@ def create_inference_graph(input_graph_def,
       use this list to determine the batch sizes of the cached engines, instead
       of making the decision on the fly. This is useful when we know the most
       common batch size(s) the application is going to generate.
+    use_calibration: selects the behavior when precision_mode is INT8.
```

Comment:
Please make it clear that if precision_mode is not INT8 this option will be ignored.